README.md

```diff
@@ -35,8 +35,8 @@ The following table will help you navigate the 3 projects and understand their d
 | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | <a href="https://learn.microsoft.com/en-us/shows/fluent-ui-insights/fluent-ui-insights-positioning?utm_source=github" target="_blank"><img src="ghdocs/medias/fluentui-ep01-preview.gif" alt="Watch EP01: Positioning" width="240" /></a> | <a href="https://learn.microsoft.com/en-us/shows/fluent-ui-insights/fluent-ui-insights-styling?utm_source=github" target="_blank"><img src="ghdocs/medias/fluentui-ep02-preview.gif" alt="Watch EP02: Styling" width="240" /></a> | <a href="https://learn.microsoft.com/en-us/shows/fluent-ui-insights/fluent-ui-insights-griffel?utm_source=github" target="_blank"><img src="ghdocs/medias/fluentui-ep03-preview.gif" alt="Watch EP03: Griffel" width="240" /></a> |
 
-|                                                                              EP04: Foundational APIs                                                                               |
-| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                                                          EP04: Foundational APIs                                                                                                                           |
```

Comment:
Looks like there are some formatting issues here to resolve, the only changed file should be CODEOWNERS

Reply:
@JustSlone I've tried disabling prettier and eslint and removing these changes manually but they seem to get added back whenever I save and/or commit; any suggestions?

Reply:
You can checkout the files and then commit using the flag --no-verify and that should stop changing them.

Reply:
this looks like something might be wrong with your dev setup. those files should not be touched by prettier nor lint as you didn't touch them.

Reply:
all of these should be reverted, ty

Reply:
I got it to work, thanks for the tip!